app/Http/Controllers/Api/ApiController.php

```diff
@@ -147,6 +147,9 @@ protected function respondForbidden($message = 'Forbidden')
      */
     protected function respondNotFound($message = 'Not Found')
     {
+        $token = $this->oauthToken();
```

Comment:
OAuth token is a credential. Make sure not to expose the value to users or logs. 
Use `->filteredOAuthToken()` for obfuscated token.